docs/build/identity.rs/1.1/docs/how-tos/verifiable-credentials/selective-disclosure.mdx

```diff
@@ -0,0 +1,138 @@
+---
+sidebar_label: Selective Disclosure
+description: Explain VC with selective disclosure.
+image: /img/Identity_icon.png
+keywords:
+  - verifiable
+  - credentials
+  - SD-JWT
+  - Disclosure
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Selective Disclosure (SD-JWT)
+
+
+Holders of verifiable credentials may prefer not to disclose all the information contained within the credential to a verifier. Instead, they may opt to only share a specific subset of the properties included in the VC. The identity library implements the [IETF Specifications](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-07.html) which outlines a mechanism to enable the selective disclosure of individual properties within the JSON object of JWT claims.
+
+## Concept
+
+#### Issuance ✍️
+During the issuance process, the issuer replaces a subset of the fields contained in a credential with digests of their salted values and creates a signed JWT. Next, JWT alongside the plain text disclosures and the salt used the for digest creation are sent to the holder. 
+
+#### Presentation 📃
+At this stage, the holder has the flexibility to selectively choose which fields to disclose to the verifier. The chosen disclosures are sent in plain text together with the JWT that contains the digests to the verifier. 
+
+:::note
+Only values replaced by digests through the issuer can be selectively disclosed. The holder **can not** conceal values provided in plain text in the JWT claims.
+:::
+
+
+#### Validation 🔬
+With these values and a valid signature, the verifier is able to reconstruct a verified VC that exclusively contains the information the holder intended to disclose.
+
+## How it works
+
+As the SD JWT remains a valid JWT, giving the following JWT claim of an address credential in accordance with the [VC Data Model v1.1](https://www.w3.org/TR/vc-data-model/#json-web-token):
```

Comment:
typo: `accordance`

Reply:
fixed

Reply:
@lucas-tortora what do you think?

Reply:
How about:
```suggestion
The following JWT represents an address credential that was issued to Alice in accordance with the [VC Data Model v1.1](https://www.w3.org/TR/vc-data-model/#json-web-token):
```
If I understood correctly, the following JSON is a plain text VC issued to Alice. In the next paragraph, you describe how she can disclose only a subset of it to keep her data as her own. Right?

Reply:
No, this is a part of the issuance step. The provided JSON are the JWT **claims** these will be made selectively disclosable as by the next step, and then will be encoded and signed as a JWT. I didn't add this information here because a previous knowledge of JWT is required, hence the sentence "As the SD JWT remains a valid JWT"

Reply:
Maybe
```suggestion
A SD JWT can be constructed from the following JWT claim of an address credential in accordance with the [VC Data Model v1.1](https://www.w3.org/TR/vc-data-model/#json-web-token):
```